source/grid/tria.cc

```diff
@@ -11887,6 +11887,9 @@ Triangulation<dim, spacedim>::create_triangulation(
               cell->face(pair.first)->set_boundary_id(pair.second);
         }
     }
+
+  // inform all listeners that the triangulation has been created
+  signals.create();
```

Comment:
The points you commented on are part of #14866, and I forked this branch from this one instead of master. I will apply your suggestions to the corresponding PR, and clean up this one.